react-native/react/constants/tracker.js

```diff
@@ -1,13 +1,21 @@
 /* @flow */
 
 // Simple state of the overall proof result
-export type SimpleProofState = 'normal' | 'warning' | 'error' | 'pending'
+export type SimpleProofState = 'normal' | 'warning' | 'error' | 'pending' | 'deleted' | 'checking'
```

Comment:
proof state and meta are different that what you have I think. was yours driven by the api state? If so we'll need to talk to coyne to reconcile that vs the design